src/jit/gschecks.cpp

```diff
@@ -473,9 +473,9 @@ void Compiler::gsParamsToShadows()
             dst = gtNewOperNode(GT_ADDR, TYP_BYREF, dst);
 
             opAssign = gtNewCpObjNode(dst, src, clsHnd, false);
-#if FEATURE_MULTIREG_STRUCTS
-            lvaTable[shadowVar].lvDontPromote = lvaTable[lclNum].lvDontPromote;
-#endif // FEATURE_MULTIREG_STRUCTS
+#if FEATURE_MULTIREG_ARGS_OR_RET
+            lvaTable[shadowVar].lvIsMultiRegArgOrRet = lvaTable[lclNum].lvIsMultiRegArgOrRet;
```

Comment:
I'm not sure what you are asking. I removed the lvDontPromote flag as it was not informative. In general, there are many reasons we decide not to promote or enregister, so I was not in favor of having a separate flag, lvDontPromote, that did not correlate with all of those reasons. So I changed the flag to be more indicative of the semantic.